__tests__/server/server.js

```diff
@@ -45,15 +45,23 @@ describe('Api test', () => {
   let server;
   beforeEach(() => {
     process.env.NODE_ENV = 'test';
+    process.env.SERVER_API_KEY = '79b2d5eb-c1fd-42c4-9391-6b2c9780d591';
```

Comment:
I am not sure it is the best idea to set this for tests via env vars, better via `config/test.json`.